backend/entities/feedback_entity.py

```diff
@@ -0,0 +1,100 @@
+"""Definition of SQLAlchemy table-backed object mapping entity for Feedback."""
+
+from sqlalchemy import Column, Integer, String, Boolean, ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from backend.models.feedback import Feedback
+from backend.models.feedback_details import FeedbackDetails
+from .entity_base import EntityBase
+from typing import Self
+from ..models.user import User
+
+
+__authors__ = ["Matt Vu"]
+__copyright__ = "Copyright 2023"
+__license__ = "MIT"
+
+
+class FeedbackEntity(EntityBase):
+    """Serves as the database model schema defining the shape of the `Feedback` table"""
+
+    # Name for the feedback table in the PostgreSQL database
+    __tablename__ = "feedback"
+
+    # Unique ID for the feedback
+    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
+    # User who submitted the feedback ticket
+    user_id = mapped_column(Integer, ForeignKey("user.id"))
+    user = relationship("UserEntity")
```

Comment:
Will we need a back-populating field here, such as if students need to see the feedback they have submitted? Regardless, does not seem to be a major concern for now.